src/backend/tests/factories/crawler_factory.py

```diff
@@ -38,5 +38,5 @@ def create_ambiguous_record_with_questions(
 
             aliases = (Alias(first_name="John", last_name="Doe", middle_name="", birth_date=""),)
             return RecordCreator.build_record(
```

Comment:
```suggestion
            record = RecordCreator.build_record(
                RecordCreator.build_search_results, "username", "password", aliases, {}
            )
            RecordCreator.search_cache = LRUCache(4) # clear the cache for each test case
            return record
```

Reply:
well. Actually that will probably break the test case that tests that the cache works.... what a finicky issue.